src/commands/workspace/inject.ts

```diff
@@ -92,10 +98,23 @@ export default class Inject extends Command {
     })
   }
 
-  async injectKubeconfigTasks(flags: any, chenamespace: string, workspace: string, container?: string): Promise<Listr> {
+  async injectKubeconfigTasks(flags: any): Promise<Listr> {
```

Comment:
Do we really need to use any?

Reply:
Yes, I think. And what's your opinion @AndrienkoAleksandr?